docs/Guidelines(8th Edition).xml

```diff
@@ -45,6 +45,7 @@
   <guideline key="Ch09_cc7b10e" severity="AVOID" section="Coding" subsection="Enums">"AVOID creating enums that represent an “incomplete” set of values, such as product version numbers."</guideline>
   <guideline key="Ch09_bf80755" severity="AVOID" section="Coding" subsection="Enums">AVOID creating flag enums where the zero value has a meaning other than “no flags are set.”</guideline>
   <guideline key="Ch09_a2194dc" severity="AVOID" section="Coding" subsection="Enums">AVOID enums that contain a single value.</guideline>
+  <guideline key="" severity="AVOID" section="Coding" subsection="Enums">DO make the enum name itself to be singular (unless the enums are bit flags).</guideline>
```

Comment:
Something looks off here. The `severity` attribute should typically match the first word of the text.

Reply:
Thanks for catching that